tests/bpf2c_tests/bpf_test.cpp

```diff
@@ -39,16 +39,6 @@ memfrob(uint64_t a, uint64_t b, uint64_t c, uint64_t d, uint64_t e)
 static uint64_t
 trash_registers(uint64_t a, uint64_t b, uint64_t c, uint64_t d, uint64_t e)
```

Comment:
Is this function still needed?  It doesn't look like it does what the function is named, with this change.